### PR TITLE
proposed changes start2finish log part 2

### PR DESCRIPTION
**Pull request by trinetta**
Things awaiting resolution:

**SIR model projections**
- Part 2: throws warnings, see git Issues, working through line by line

Things that run:
**Automated data collection and processing**

- China

**SIR model projections**
- Part 1 `python codes/models/get_gamma.py`

**Regression model estimation**
**Figure Creation** 
- Fig1
- Fig2
- Fig3
- Fig4
- Appendix Table A1
- Appendix Figure A1
- Appendix Figure A2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -261,4 +261,4 @@ Figure A1 is generated by the regression estimation step (`codes/models/alt_grow
 #### Appendix Figure A2
 
 1. `Rscript codes/data/korea/make_JHU_comparison_data.R`: Creates [data/interim/korea/KOR_JHU_data_comparison.csv](data/interim/korea/KOR_JHU_data_comparison.csv)
-2. `python codes/plotting/figA2.py`: Generates 2 outputs that constitute Figure A2 (`results/figures/appendix/figA2-1.pdf` and `results/figures/appendix/figA2-2.pdf`)
```
Comment (bolliger32) [inline README.md]: I changed all figures and tables to go in a results folder. Will be helpful if we make this repo a [Code Ocean](https://codeocean.com/) capsule which i think would be cool and very little marginal extra work once the repo is clean. Can you undo this change?

File: codes/plotting/count-policies.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 path_data_sources = cutil.HOME / 'references' / 'data_sources_static_20200321.xlsx'
 
-path_out_csv = cutil.HOME / 'results' / 'tables' / 'table_a1' / 'policy_counts_table_raw.csv'
```
Comment (bolliger32) [inline codes/plotting/count-policies.py]: same here

File: codes/plotting/fig1.R [hunk 3]
```diff
@@ -11,7 +11,7 @@ library(magrittr)
 library(ggplot2)
 
 data_dir <- "data/"
-output_dir <- "results/figures/fig1/"
```
Comment (bolliger32) [inline codes/plotting/fig1.R]: same here

File: codes/plotting/figA2.py [hunk 4]
```diff
@@ -8,7 +8,7 @@
 matplotlib.rcParams['pdf.fonttype'] = 42
 matplotlib.rcParams['axes.linewidth'] = 2
 
-out_dir = cutil.HOME / 'results' / 'figures' / 'appendix'
```
Comment (bolliger32) [inline codes/plotting/figA2.py]: same here

File: codes/plotting/gen_fig4.py [hunk 5]
```diff
@@ -10,7 +10,7 @@
 
 # save the figure here
 save_fig = True
-fig_dir = cutil.HOME / 'results' / 'figures' / 'fig4'
```
Comment (bolliger32) [inline codes/plotting/gen_fig4.py]: same here

**Comment by trinetta** [comment]
got rid of all the commits related to deleting `results/figures/` and `results/tables`!

File: README.md [hunk 6]
```diff
@@ -256,7 +256,7 @@ Note that the outputs of [codes/plotting/fig1.R](codes/plotting/fig1.R) are requ
 
 #### Appendix Figure A1
 
-Figure A1 is generated by the regression estimation step (`codes/models/alt_growth_rates/MASTER_run_all_reg.do`)
+Figure A1 is generated by the regression estimation step (`codes/models/alt_growth_rates/MASTER_run_all_reg.do`). The final output file is `figures/appendix/ALL_conf_cases_e.png`. Please note that if you're running the Stata console in Unix, .png file formats are not supported and you would need to change the final format in line 50 of `codes/models/alt_growth_rates/MASTER_run_all_reg.do` from .png to either .eps or .ps. For more information on supported file formats while using the `graph export` command on different operating systems, please click [here](https://www.stata.com/manuals13/g-2graphexport.pdf).
```
Comment (bolliger32) [inline README.md]: gross! good catch.
